docs/advanced/exporters.md

```diff
@@ -0,0 +1,61 @@
+# About the exporters
+
+The mbed exporters are used to export your code to various third party tools and IDEs. Each exporter implements a `generate` function that produces an IDE specific project file. Exporters benefit from [mbed build tools](https://github.com/ARMmbed/mbed-os/blob/master/docs/BUILDING.md#build-mbed-sdk-library-from-sources). However, instead of using your source and [config data](https://github.com/ARMmbed/mbed-os/blob/master/docs/config_system.md) to create an executable, we use that information to populate an IDE project file that will be configured to build, flash and debug your code. You can find exporter implementations [here](https://github.com/ARMmbed/mbed-os/tree/master/tools/export).
+
+## mbed-cli command
+
+`mbed export -m [target] -i [IDE]`
```

Comment:
Is there any information about how to use the target and IDE flags? Permitted strings, can I use more than one target per command etc.